src/client/getDataFromStack.js

```diff
@@ -1,29 +1,43 @@
 import URL from '../server/url.js';
 
-export async function getData () {
-    try {
-        const response = await fetch(URL);
-        const json = await response.json();
 
-        let tableHtml = '';
+function wrapStackObject(json){
+    let tableHtml = '';
+    for (const code in json) {
+        let massTags = json[code].tags;
+        let canAdd = '&#10006';
 
-        for (const code in json) {
-            tableHtml += `
-               <tr>
-               <td>${code} </td>
-               <td>${json[code].question_id} </td>
-               <td>${json[code].tags} </td>
-               <td>${json[code].title} </td>
-               <tr>
-               `;
+        if(massTags.length == 5){
+            canAdd = '&#10004';
+        }
+        let tagsString = ``;
+        for(const idTag in massTags){
+             tagsString+= `<td class = "chip" style ="text-allign:center">${massTags[idTag]} </td>\n`
+        }
+    
+        tableHtml += `
+           <tr align ="center">
+           <td class = "hoverable" id = ${code}>${code} </td>
+           <td class = "hoverable" id  = "idDataTable">${json[code].question_id} </td>
+           ${tagsString}
+           <td class = "hoverable">${json[code].title} </td>
+           <td class = "hoverable">${canAdd}</td>
+           <tr>
+           `;
         }
+    return tableHtml;
+}
 
+export async function getData () {
+    try {
+        const response = await fetch(URL);
+        const json = await response.json();
+        console.log(json);
```

Comment:
- Please add `debug` to the import map in your HTML file and use it instead
 - Modify the `lint` task to cover all project files